File: backend/PswProject/PswProject/dto/CommentDTO.cs
using System;

namespace PswProject.dto
{
    public class CommentDTO
    {
        public String Name { get; set; }
        public DateTime Date { get; set; }
        public String Content { get; set; }
        public int Rating { get; set; }

        public CommentDTO()
        {

        }
    }
}

File: frontend/psw-frontend/src/app/pages/comment/comment.dto.ts
export class CommentDTO{
    Name: string;
    Content: string;
    Date: Date;
    Rating: any;
    CanPublish: boolean;
    isVisible: boolean;
    constructor(){
        this.Name = "",
        this.Content = "",
        this.Date = new Date(),
        this.Rating = 0,
        this.CanPublish = true,
        this.isVisible = false
    }
}